bundles/pubsub/test/test/test_endpoint_runner.cc

```diff
@@ -18,22 +18,54 @@
  */
 
 #include "celix_api.h"
```

Comment:
We are moving to GTest, not sure if it makes sense to do this now as well.

Reply:
The TcpAdmin tests already exists for more then 6 months. When the pubsub tests are converted to gtest. These can be converted to gtest. This test tests the tcp static endpoint functionality and is variant of the pubsub unit tests.